monkey/monkey_island/cc/next_ui/src/providers/authProvider/provider.tsx

```diff
@@ -25,7 +24,8 @@ export function AuthProvider({ children }: { children: ReactNode }) {
         const currentPathName: string = window.location.pathname;
         isSessionHasError(data);
         if (
-            (status === UNAUTHENTICATED || isSessionHasError(data)) &&
+            (status === AUTH_STATUS.UNAUTHENTICATED ||
+                isSessionHasError(data)) &&
             !isPathInAuthPaths(currentPathName)
```

Comment:
We should check if the path is public. There might be more public paths in the future

Reply:
Currently, we don't have any public pages. When will get to the point where will have them, we'll change it accordingly

Reply:
I think it's better to do `isPathPrivate` instead of `!isPathInAuthPaths` here. In any case, we want to check if the path is not **public** here, not if the path is not an authentication path.